ports/osqp-eigen/vcpkg.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "osqp-eigen",
+  "version": "0.7.0",
+  "port-version": 2,
+  "description": "Simple Eigen-C++ wrapper for OSQP library",
+  "homepage": "https://robotology.github.io/osqp-eigen/",
+  "dependencies": [
+    {
+      "name": "eigen3",
+      "version>=": "3.4.0"
+    },
+    {
+      "name": "osqp",
+      "version>=": "0.6.2"
+    },
```

Similar to eigen. Dut assuming a tight coupling between osqp and osqp-eigen, it may make sense to leave that version constraint.
```suggestion
    "osqp",
```

---

Yes, there is indeed a tight coupling, so it would be good to leave this constraint in place.